public/v4/apps/cloudflare-tunnel.yml

```diff
@@ -0,0 +1,25 @@
+captainVersion: 4
+services:
+    $$cap_appname:
+        caproverExtra:
+            notExposeAsWebApp: 'true'
+            dockerfileLines:
+                - FROM cloudflare/cloudflared:latest
```

Comment:
Please use non `latest` tags. Also, it's a good practice to expose the version in the variables .